miniapp/src/main/assets/bridge.js

```diff
@@ -91,14 +94,30 @@ MiniAppBridge.getUniqueId = function() {
   return new Promise(function(resolve, reject) {
     return MiniAppBridge.exec(
       "getUniqueId",
+      null,
```

Comment:
What do you think of passing an empty payload instead of null?

Reply:
`null` represents the non-usage so I think null is better.